.github/workflows/tests.yml

```diff
@@ -151,12 +155,12 @@ jobs:
             core.setOutput('container', JSON.stringify(container))
 
             const test_bot_formulae_args = ["--only-formulae", "--junit", "--only-json-tab", "--skip-dependents"]
-            test_bot_formulae_args.push('--testing-formulae=${{needs.tap_syntax.outputs.testing_formulae}}')
-            test_bot_formulae_args.push('--added-formulae=${{needs.tap_syntax.outputs.added_formulae}}')
-            test_bot_formulae_args.push('--deleted-formulae=${{needs.tap_syntax.outputs.deleted_formulae}}')
+            test_bot_formulae_args.push(`--testing-formulae=${process.env.TESTING_FORMULAE}`)
+            test_bot_formulae_args.push(`--added-formulae=${process.env.ADDED_FORMULAE}`)
+            test_bot_formulae_args.push(`--deleted-formulae=${process.env.DELETED_FORMULAE}`)
```

Comment:
Do we want to add quotes here to make sure the contents of these environment variables aren't interpreted by the shell?

Reply:
Makes sense